action/protocol/staking/handlers.go

```diff
@@ -67,85 +67,92 @@ func (h *handleError) ReceiptStatus() uint64 {
 }
 
 func (p *Protocol) handleCreateStake(ctx context.Context, act *action.CreateStake, csm CandidateStateManager,
-) (action.Topics, error) {
-	topics := action.Topics{hash.BytesToHash256([]byte(HandleCreateStake))}
+) (*receiptLog, error) {
 	actionCtx := protocol.MustGetActionCtx(ctx)
 	blkCtx := protocol.MustGetBlockCtx(ctx)
+	postFairbankMigration := blkCtx.BlockHeight >= p.hu.FbkMigrationBlockHeight()
+	log := newReceiptLog(p.addr.String(), HandleCreateStake, postFairbankMigration)
```

Comment:
maybe put 
```
postFairbankMigration := blkCtx.BlockHeight >= p.hu.FbkMigrationBlockHeight()
```
in `newReceiptLog`?

Reply:
then need to pass height and FbkMigrationBlockHeight() into newReceiptLog(), feel that's too much

receiptLog only need  pre/post, don't need the actual height